### PR TITLE
Add support to 8.0.x tenacity version

### LINKED ISSUE #268: Dependency conflicts
Recently I noticed some errors where pytest-selenium needs tenacity older than version 7 but newer than or equal to version 6. Tenacity has a version 7.0.0. Can we change pytest-selenium to use tenacity version 7 or does it need to use a version older than 7.0.0.

### PR DESCRIPTION
**Pull request by lordgordon**
Closes #268

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -9,7 +9,7 @@ To setup the virtual environment and pre-commit, run:
 
 .. code-block:: bash
 
-  $ pipenv install --dev
+  $ pipenv install --dev --pre
```
Comment (lordgordon) [inline docs/development.rst:12]: To solve this error when first configuring the local environment:
```
Pipfile.lock not found, creating...
Locking [dev-packages] dependencies...
Building requirements...
Resolving dependencies...
✘ Locking Failed!
[ResolutionFailure]:   File "/opt/homebrew/lib/python3.9/site-packages/pipenv/resolver.py", line 741, in _main
[ResolutionFailure]:       resolve_packages(pre, clear, verbose, system, write, requirements_dir, packages, dev)
[ResolutionFailure]:   File "/opt/homebrew/lib/python3.9/site-packages/pipenv/resolver.py", line 702, in resolve_packages
[ResolutionFailure]:       results, resolver = resolve(
[ResolutionFailure]:   File "/opt/homebrew/lib/python3.9/site-packages/pipenv/resolver.py", line 684, in resolve
[ResolutionFailure]:       return resolve_deps(
[ResolutionFailure]:   File "/opt/homebrew/lib/python3.9/site-packages/pipenv/utils.py", line 1395, in resolve_deps
[ResolutionFailure]:       results, hashes, markers_lookup, resolver, skipped = actually_resolve_deps(
[ResolutionFailure]:   File "/opt/homebrew/lib/python3.9/site-packages/pipenv/utils.py", line 1108, in actually_resolve_deps
[ResolutionFailure]:       resolver.resolve()
[ResolutionFailure]:   File "/opt/homebrew/lib/python3.9/site-packages/pipenv/utils.py", line 833, in resolve
[ResolutionFailure]:       raise ResolutionFailure(message=str(e))
[pipenv.exceptions.ResolutionFailure]: Warning: Your dependencies could not be resolved. You likely have a mismatch in your sub-dependencies.
  First try clearing your dependency cache with $ pipenv lock --clear, then try the original command again.
 Alternatively, you can use $ pipenv install --skip-lock to bypass this mechanism, then run $ pipenv graph to inspect the situation.
  Hint: try $ pipenv lock --pre if it is a pre-release dependency.
ERROR: Could not find a version that matches black (from -r /var/folders/tn/53xwmk6x7jgfxhjzhq1kjy3h0000gn/T/pipenvkxo3nalyrequirements/pipenv-49n2nkci-constraints.txt (line 6))
Skipped pre-versions: 18.3a0, 18.3a0, 18.3a1, 18.3a1, 18.3a2, 18.3a2, 18.3a3, 18.3a3, 18.3a4, 18.3a4, 18.4a0, 18.4a0, 18.4a1, 18.4a1, 18.4a2, 18.4a2, 18.4a3, 18.4a3, 18.4a4, 18.4a4, 18.5b0, 18.5b0, 18.5b1, 18.5b1, 18.6b0, 18.6b0, 18.6b1, 18.6b1, 18.6b2, 18.6b2, 18.6b3, 18.6b3, 18.6b4, 18.6b4, 18.9b0, 18.9b0, 19.3b0, 19.3b0, 19.10b0, 19.10b0, 20.8b0, 20.8b1, 21.4b0, 21.4b0, 21.4b1, 21.4b1, 21.4b2, 21.4b2, 21.5b0, 21.5b0, 21.5b1, 21.5b1, 21.5b2, 21.5b2, 21.6b0, 21.6b0, 21.7b0, 21.7b0
There are incompatible versions in the resolved dependencies:
```

File: Pipfile [hunk 2]
```diff
@@ -15,3 +15,6 @@ pytest-mock = "*"
 
 [packages]
 pytest-selenium = {editable = true,extras = ["appium"],path = "."}
+
+[pipenv]
+allow_prereleases = true
```
Comment (lordgordon) [inline Pipfile:20]: automatically added when running `pipenv install --dev --pre`

File: pytest_selenium/hooks.py [hunk 3]
```diff
@@ -4,8 +4,8 @@
 
 
 def pytest_selenium_capture_debug(item, report, extra):
-    """ Called when gathering debug information for the HTML report. """
+    """Called when gathering debug information for the HTML report."""
```
Comment (lordgordon) [inline pytest_selenium/hooks.py:7]: some cleanup made by black

**Review by isaulv — APPROVED** [review]
LGTM.

**Comment by lordgordon** [comment]
@BeyondEvil thanks for migrating to Poetry. With the migration, this PR is now obsolete and the original issue is fixed.

**Pull request closed without merging**